pkg/controller/vsphere/util_test.go

```diff
@@ -0,0 +1,57 @@
+package vsphere
+
+import (
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestGetVSphereConfig(t *testing.T) {
+	testConfig := `
+    [Labels]
```

Comment:
is this a leftover bit or something? i don't recognize the syntax here.

Reply:
That's INI config, that comes from cloud `cloud-provider-config` config map. Had to learn that too :)

Reply:
oh, interesting, guess i need to learn it as well XD

Reply:
Who chose INI XD